src/login/scss/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$main-color: #6699ff;
+$main-darknes: #3366ff;
+$white-color: white;
```

Comment:
Colors are recognized differently in browsers. So it's better to keep them as code. Like #fff here.